style.css

```diff
@@ -332,3 +336,89 @@ header a {
   margin: 0 24px;
   padding-left: 0;
 }
+
+/* contact section */
+.contact-section {
+  display: flex;
+  width: 100%;
+  justify-content: left;
+  background: white;
+  padding: 0;
+  flex-direction: column;
+  align-items: center;
+}
+
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+.contact-section h2 {
+  font-family: var(--font-inter);
+  font-weight: 700;
+  color: #172b4d;
+  text-align: center;
+  font-size: 2em;
+  padding-top: 15vh;
+  margin-bottom: 10vh;
+}
+
+.contact-form {
+  margin-bottom: 10vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-form input {
+  width: 100%;
+  border: none;
+  border-bottom: 0.5px solid lightgray;
+  padding: 5vh 0 2vh 2vh;
+  margin-bottom: 2vh;
+}
+
+#error-message {
+  color: red;
+  display: block;
+}
+
+.contact-form textarea {
+  width: 98%;
+  height: 20vh;
+  border: none;
+  background: #fbf8f7;
+  margin: 2vh 0;
+  padding: 2vh 2vh 0;
+}
+
+.submit-btn {
```

Comment:
- [ ] Kindly adhere to the design as I mentioned above which makes your design looks perfect and good to *eyes*. Actually building the design exactly the same as the figma `UI`, actually tell any person how much you are `detail-oriented` and indeed helps you to be the more professional developer who takes care of the `clients` requirements. Please have a look at the below image and one last thing make sure that your text looks exactly the same as per the requirement, you just need to add the `border: none;` property to match your button design, but don't forget to look for the `font-family` and colour too.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-10 17-03-43](https://github.com/ClarenceAtim/Portfolio/assets/66476329/3d5bed37-d21b-41d2-ad16-4f7776a4c61d)

</td>
<td>

![Screenshot from 2023-05-10 17-13-39](https://github.com/ClarenceAtim/Portfolio/assets/66476329/7185131f-c561-4948-a275-98e1c6b07cc7)

</td>
</tr>
</table>


### Reminder

![image](https://github.com/geekelo/a-personal-portfolio-site/assets/66476329/c43ee2e3-782e-4080-875a-b7311226d8fa)